layers/+spacemacs/spacemacs-evil/packages.el

```diff
@@ -138,6 +138,9 @@
                   lazy-load-evil-escape-2)))
     :config
     (add-hook 'spacemacs-editing-style-hook #'spacemacs//evil-escape-deactivate-in-holy-mode)
+    (add-hook 'evil-normal-state-entry-hook (lambda ()
+                                              (when (and (fboundp 'company--active-p) (company--active-p))
```

Comment:
I would suggest instead of using `(fbound 'company--active-p)` to wrap the thing in a `(with-eval-after-load 'company ...)` so that the hook is only added when company gets loaded.

Reply:
Thanks for the hint. I'll incoporate using `with-eval-after-load`, but I would still check if `company--active-p` is bound, because it is not a public function and thus could be removed or renamed, which would then cause an error and be a bad UX. What do you think?

Reply:
Both @dalanicolai's and your point are compatible, if I'm not wrong. I see benefit of not adding the hook at all until company gets loaded.